integration_testing/features/coach/coach-create-quizzes.feature

```diff
@@ -1,50 +1,41 @@
-Feature: Coach create quizzes
+Feature: Coach creates quizzes
   Coach needs to be able to create quizzes from existing content
 
   Background:
     Given I am signed in to Kolibri as a coach user
-    And I am on the *Coach > Quizzes* page
+      And I am on the *Coach > Plan > Quizzes* page
 
   Scenario: Create new quiz
     When I click the *New quiz* button
     Then I see a new *Create new quiz* page
-      And I see empty *Title* and *Number of questions* fields
-      And I see *0 total selected* exercises
+      And I see empty *Title* field, and the default value of 10 in the *Number of questions* field
       And I see a list of channels that contain exercises
-      But I don't see any checkboxes
+        But I don't see any checkboxes
 
   # Given there are no channels that have exercises
   # Then there should not be any channels available to select
   Scenario: Try and fail to create new quiz when no channel on device contains exercises
     When I click the *New quiz* button
     Then I see a new *Create new quiz* page
       And I see empty *Title* and *Number of questions* fields
-      And I see *0 total selected* exercises indicator
       But I don't see any channels
 
-  Scenario: Check validation for the title field
-    Given I have not inputted an quiz title (yet)
-    When I leave the input field or attempt to preview or save the quiz
-    Then an input validation error appears
-    When I input a valid title into the field
-    Then I don't see the validation error anymore
-
   Scenario: Check validation for the number of questions field
     When I input a number outside the range of 1-50
-      And I leave the input field or attempt to preview or save the quiz
-    Then an input validation error appears
+      And I leave the input field or attempt to continue or finish the quiz
+    Then the input field has the red outline
     When I input a number greater than amount of exercise questions
-      And I leave the input field or attempt to preview or save the quiz
-    Then an input validation error appears
+      And I leave the input field or attempt to continue or finish the quiz
+    Then the number is cut out at 50
     When I don't input anything into the number field
-      And I leave the input field or attempt to preview or save the quiz
+      And I leave the input field or attempt to continue or finish the quiz
     Then an input validation error appears
+    # This is not working right, quiz is saved even if the number of questions is left empty
```

Comment:
@indirectlylit Need to file an issue here or is it in the works? ^^

Reply:
created an issue here: https://github.com/learningequality/kolibri/issues/4932

we should also add gherkin stories that have us trying to create quizzes and lessons with duplicate names, and getting validation errors for those. (https://github.com/learningequality/kolibri/pull/4777)

Reply:
Roger, will add! 👍